src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/RestoreOperation/RestoreDatabaseTaskDataObject.cs

```diff
@@ -247,6 +274,37 @@ private void RelocateDbFiles()
             }
         }
 
+        /// <summary>
+        /// Updates the Restore folder location of those db files whose orginal directory location
+        /// is not present in the destination computer.
+        /// </summary>
+        internal void UpdateDBFilesPhysicalRelocate()
+        {
+            foreach (DbFile item in DbFiles)
+            {
+                string fileName = this.GetTargetDbFilePhysicalName(item.PhysicalName);
+                item.PhysicalNameRelocate = PathWrapper.Combine(PathWrapper.GetDirectoryName(item.PhysicalName),
+                    fileName);
+                Uri pathUri;
+                bool fUriCreated = Uri.TryCreate(item.PhysicalNameRelocate, UriKind.Absolute, out pathUri);
```

Comment:
we shouldn't use Hungarian notation for variables...please remove the 'f' prefix.